Dockerfile

```diff
@@ -28,8 +28,8 @@ USER node
 # Can be a tag, release, but prefer a commit hash because it's not changeable
 # https://github.com/bitwarden/clients/commit/${VAULT_VERSION}
 #
-# Using https://github.com/bitwarden/clients/releases/tag/web-v2022.10.0
-ARG VAULT_VERSION=feb10128f0963bf6a9dabf23d1a01d8ae8ea117c
+# Using https://github.com/bitwarden/clients/releases/tag/web-v2022.10.1
+ARG VAULT_VERSION=ae435905f7758262694a78fca54b799b5e7be0a5
```

Comment:
Isn't the commit rather `02ae7ad9b3eb01051295e8a2ca627cb4773b424a`? At least in the master branch?

The commit `ae435905f7758262694a78fca54b799b5e7be0a5` is part of [hotfix-rc-web](https://github.com/bitwarden/clients/compare/hotfix-rc-web). I am just wondering what happens, if that branch is gone.

Reply:
ae435905f7758262694a78fca54b799b5e7be0a5 is simply the commit hash of the release [web-v2022.10.1 ](https://github.com/bitwarden/clients/releases/tag/web-v2022.10.1).

Reply:
A git hash is only created for a commit (and also for a merge commit, and squashing will also create new git hashes). Tagging does not create a commit hash, but adds a tag to a commit.

It seems that the tag was added to a commit in the branch `hotfix-rc-web`. However, that tag does not seem to exist in the master branch.

I am just saying that if someone were to delete the branch `hotfix-rc-web`, you could not reference this commit point with the git hash that is in that branch, because it would be gone.

But maybe I completely misunderstood how git works.

Reply:
As far as i know the hashes from the tagges keep existing. Someone could overwrite a tag if course and then it could probably fall.

Also, if you take the commit hash from master, then you will also get a bunch of other changes which should not be part of that specific release, like new upcoming features for example, because the commit chain there is different.

So, this is the correct hash 😃.

@stefan0xC could you also make sure that `DuckDuckGo` is hidden from the `Forwarded Alias` list? Same as is done for FireFox already. It doesn't work for self-hosted and only generates errors and issue reports.

Reply:
@BlackDex I made a separate PR to address this issue.